hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/TopicContentTypeMigrationService.java

```diff
@@ -17,8 +21,9 @@
 
     private static final Logger logger = LoggerFactory.getLogger(TopicContentTypeMigrationService.class);
 
-    public static final Duration CHECK_OFFSETS_AVAILABLE_TIMEOUT = Duration.ofSeconds(1);
-    public static final Duration INTERVAL_BETWEEN_OFFSETS_AVAILABLE_CHECK_RETRIES = Duration.ofMillis(500);
+    private static final Duration CHECK_OFFSETS_AVAILABLE_TIMEOUT = Duration.ofSeconds(30);
+    private static final Duration INTERVAL_BETWEEN_OFFSETS_AVAILABLE_CHECK = Duration.ofMillis(500);
+    private static final Duration INTERVAL_BETWEEN_ASSIGNMENTS_COMPLETED_CHECK = Duration.ofMillis(500);
```

Comment:
can we have these 3 params configurable?

Reply:
Currently we have not any configuration for that, if something goes wrong user can perform retransmission one more time. If we will need this in future we will make different pull request :)

Reply:
👍